addons/resource.language.en_gb/resources/strings.po

```diff
@@ -13672,7 +13672,12 @@ msgctxt "#20473"
 msgid "Premiered"
 msgstr ""
 
-#empty strings from id 20474 to 21329
+#. xbmc/playlists/SmartPlayList.cpp
```

Comment:
Use `#:` to reference the file where the string is used, not `#.`
`#.` is for verbal explanation of the purpose and context of this string. translators really need this information. So, for new strings always add `#.`

Reply:
Thanks.  Done and comment added for translators.

Reply:
Thank you.